pkg/cloudprovider/providers/azure/azure.go

```diff
@@ -264,17 +259,44 @@ func NewCloud(configReader io.Reader) (cloudprovider.Interface, error) {
 			config.CloudProviderRateLimitBucketWrite)
 	}
 
+	// Conditionally configure resource request backoff
+	if config.CloudProviderBackoff {
+		// Assign backoff defaults if no configuration was passed in
+		if config.CloudProviderBackoffRetries == 0 {
+			config.CloudProviderBackoffRetries = backoffRetriesDefault
+		}
+		if config.CloudProviderBackoffDuration == 0 {
+			config.CloudProviderBackoffDuration = backoffDurationDefault
+		}
+		// Log warning for deprecated options.
+		if config.CloudProviderBackoffExponent != 0 {
+			klog.Warning("Azure cloud provider config 'cloudProviderBackoffExponent' has been deprecated. 2 is always used as the backoff exponent.")
+		}
+		if config.CloudProviderBackoffJitter != 0 {
+			klog.Warning("Azure cloud provider config 'cloudProviderBackoffJitter' has been deprecated. 2 is always used as the backoff exponent.")
```

Comment:
This error is incorrect (copy and pasted...) and also, I think we still need to obey the user provided value.

Reply:
oops, my mistake